src/plugins/themes.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const requireg = require("requireg");
+var colors = require("colors/safe");
+const themes = {};
+
+module.exports = {
+	get: get,
+	fileName: fileName
+};
+
+fs.readdir("client/themes/", (err, dirThemes) => {
+	if (err) {
+		return;
+	}
+	dirThemes
+		.filter(endsWithCss)
+		.map(makeThemeObject)
+		.forEach((theme) => themes[theme.name] = theme);
+});
+
+Helper.config.themes.map(getModuleInfo)
+	.filter((name) => name !== undefined)
+	.forEach((theme) => themes[theme.module] = theme);
+
+function get() {
+	return Object.keys(themes).map((key) => themes[key]);
+}
+
+function fileName(module) {
+	return themes[module].filename;
+}
+
+function endsWithCss(themeFile) {
+	return themeFile.endsWith(".css");
+}
+
+function makeThemeObject(css) {
+	const filename = css.slice(0, -4);
+	return {
+		name: filename.charAt(0).toUpperCase() + filename.slice(1),
+		filename: filename,
+		url: `/themes/${filename}.css`
+	};
+}
+
+function getModuleInfo(moduleName) {
+	let module;
+	try {
+		module = requireg(moduleName);
+	} catch (e) {
+		log.warn(`Specified theme ${colors.yellow(moduleName)} is not installed globally`);
+		return;
+	}
+	const modulePath = requireg.resolve(moduleName);
+	modulePath.substring(0, modulePath.lastIndexOf("/"));
+	return {
+		module: moduleName,
+		name: module.name,
+		filename: `${modulePath.substring(0, modulePath.lastIndexOf("/"))}/${module.css}`,
```

Comment:
> Also before returning, make sure this file actually exists. Use path.join and path.normalize.

What should it do if the file doesn't exist? Return undefined? Return an object with no filename? Return something that will show an error in the frontend?